src/main/resources/templates/smallfull/approval.html

```diff
@@ -10,16 +10,115 @@
 <body>
 <div id="approval-main-content" layout:fragment="content">
 
-  <form id="approval-form" th:action="@{''}" class="form" method="post">
+  <form id="approval-form" th:action="@{''}" th:object="${approval}" class="form" method="post">
 
-    <div th:replace="fragments/numberedHeading :: numberedHeading (
-                    number = '5',
-                    headingText = 'Approve your accounts'
-                )">
-    </div>
+    <div th:replace="fragments/globalErrors :: globalErrors"></div>
+
+    <div class="govuk-grid-row">
+
+      <div class="govuk-grid-column-two-thirds">
+
+        <fieldset class="govuk-fieldset">
+
+          <legend id="page-title" class="govuk-fieldset__legend govuk-fieldset__legend--l">
+            Approve your accounts
+          </legend>
+
+          <div class="govuk-form-group" th:classappend="${#fields.hasErrors('directorName')} ? 'govuk-form-group--error' : ''">
+            <label id="approval-director-heading" class="govuk-label govuk-label--m" for="approval-director">
+              Which director approved the accounts?
+            </label>
+
+            <span id="approval-director-hint" class="govuk-hint">
+              If the company has two or more directors, give the name of the director acting on their behalf.
+            </span>
+
+            <span class="govuk-error-message"
+                  id="directorName-errorId"
+                  th:if="${#fields.hasErrors('directorName')}"
+                  th:each="e : ${#fields.errors('directorName')}" th:text="${e}" >
+                            </span>
```

Comment:
Could you fix the alignment of the `</span>` element here and remove the trailing space before the closing `>` of the opening `<span>` element?